icon/iiss/icstate/prepstatus.go

```diff
@@ -116,9 +116,23 @@ func (ps *PRepStatus) SetDelegated(delegated *big.Int) {
 	ps.delegated.Set(delegated)
 }
 
-func (ps *PRepStatus) GetBondedDelegation() *big.Int {
-	// TODO: Not implemented
-	return ps.delegated
+func (ps *PRepStatus) GetBondedDelegation(bondRequirement int) *big.Int {
```

Comment:
Need a comment to help understand

``` go
	// totalDelegation = bond + delegation
	// bondRatio = bond / totalDelegation * 100
	// bondedDelegation = totalDelegation * (bondRatio / bondRequirement)
	//                  = bond * 100 / bondRequirement
	// if bondedDelegation > totalDelegation
	//    bondedDelegation = totalDelegation
```

Change local variable name as per the comments to improve readability

Reply:
I added the comments and changed name of variables.
Thanks.